js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -99,6 +99,21 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating requestCustomPermissions function to MiniAppBridge object
+   * @param {string[]} permissionTypes, Array of type of custom permissions that is requested
```

Comment:
Array.<string> would be better. Also please try to add an example that could be much more helpful.

Reply:
I have added an example, let me to know if it's okay, or we can do something better.

Reply:
Yeah, I think this needs an example of the string array that should be passed.